doc/engineering-reference/src/simulation-models-encyclopedic-reference-001/ice-thermal-storage.tex

```diff
@@ -103,7 +103,11 @@ \subsection{Detailed Ice Storage Model}\label{detailed-ice-storage-model}
 
 \subsubsection{Charging and Discharging Equation}\label{charging-and-discharging-equation}
 
-The actual performance of the ice storage unit depends on the physical geometry, materials, and characteristics of the ice storage unit.~ In analyzing performance data trends from ice storage manufacturers, it was determined that the following equation would work well for the discharging process:
+The actual performance of the ice storage unit depends on the physical geometry, materials, and characteristics of the ice storage unit. Modeling both the charging and discharging performance of a particular ice storage unit is accomplished in EnergyPlus using Curve objects that establish the relationships between various parameters of the unit and the output of the unit itself. The input that controls what curve objects are used and what parameters are the independent variables in those curves are: Discharing Curve Variable Specifications, Discharging Curve Name, Charging Curve Variable Specifications, and Charging Curve Name.
+
+The Variable Specifications input fields determine which parameters are used as the independent variables in the curves referenced by the Curve Name inputs.  There are four options for the Variable Specifications fields: PercentChargedLMTD, PercentDischargedLMTD, LMTDMassFlow, and LMTDPercentCharged.  Each option defines which two variables are the independent variables in the curve equation as well as the order.  For example, PercentChargedLMTD uses the percent charged or fraction of full charge of the storage component as the first variable and the normalized LMTD (or LMTD*) as the second variable.  Note that using options that use either LMTD or MassFlow actually use normalized values LMTD* and MassFlow*.  LMTD is normalized by dividing the actual value by 10\(^{\circ}\)C while MassFlow is normalized by dividing the actual mass flow rate by the equivalent of 100 gallons per minute.  
```

Comment:
Looks like you changed your mind from PercentCharged/Discharged* to FractionCharged/Discharged? The IDD and code use Fraction, but the docs are mostly Percent.

Reply:
@mjwitte Good catch.  My bad.  I have fixed this so there are no more Percent*** in this doc.